src/diffpy/structure/atom.py

```diff
@@ -536,7 +536,7 @@ def __setitem__(self, idx, value):
         self._atom.xyz[:] = self._atom.lattice.fractional(self)
         return
 
-    def __array_wrap__(self, out_arr, context=None):
+    def __array_wrap__(self, out_arr, context=None, return_scalar=None):
```

Comment:
We are overriding and the `__array_wrap__` method within `numpy.ndarray` and it requires `return_scalar` in `numpy 2.0.0` or above to remove the following warnings:

```
te-packages/numpy/_core/numeric.py:2462: DeprecationWarning: __array_wrap__ must accept context and return_scalar arguments (positionally) in the future. (Deprecated NumPy 2.0)
```